ulwgl_run.py

```diff
@@ -327,8 +328,16 @@ def main() -> None:  # noqa: D103
         os.environ[key] = val
 
     build_command(env, command, opts)
-    subprocess.run(command)
+    proc = subprocess.run(command)
+    return proc.returncode
 
 
 if __name__ == "__main__":
-    main()
+    try:
+        ret = main()
+    except Exception as e:  # noqa: BLE001
+        traceback.print_tb(e.__traceback__, file=sys.stderr)
```

Comment:
Standard error  is the default, right? Also, when using print_exception:
> print_exception(e)

Reply:
Yes, I like to be explicit sometimes with no good reason really. Removed it.